src/olympia/addons/admin.py

```diff
@@ -194,10 +194,26 @@ def change_view(self, request, object_id, form_url='', extra_context=None):
                     url += '?' + request.GET.urlencode()
                 return http.HttpResponsePermanentRedirect(url)
 
-        return super(AddonAdmin, self).change_view(
-            request, object_id, form_url, extra_context=None,
+        return super().change_view(request, object_id, form_url,
+                                   extra_context=extra_context)
+
+    def render_change_form(self, request, context, add=False, change=False,
+                           form_url='', obj=None):
+        context.update(
+            {
+                'has_listed_versions': obj.has_listed_versions(
+                    include_deleted=True
+                ),
+                'has_unlisted_versions': obj.has_unlisted_versions(
+                    include_deleted=True
+                )
+            }
```

Comment:
thankfully we don't support creating new `Addon` instances via django admin because this would fail in that case 😬 (obj would be None)

Reply:
I saw the same thing in the blocklist admin so I thought it was okay :D

Reply:
If it's https://github.com/mozilla/addons-server/blob/master/src/olympia/blocklist/admin.py#L397 note the `if change:` - `change=True` when it's not a new instance.

Reply:
ha! /me writes a followup patch